portal-db/Makefile

```diff
@@ -166,6 +166,11 @@ postgrest-swagger-ui: postgrest-generate-openapi ## Start Swagger UI to view Ope
 		-v $(PWD)/api/openapi/openapi.json:/openapi.json:ro \
 		swaggerapi/swagger-ui
 
+.PHONY: postgrest-generate-sdks
+postgrest-generate-sdks: postgrest-generate-openapi ## Generate Go and TypeScript SDKs from OpenAPI specification
```

Comment:
Can you improve the DevEx here so it's easier for the developer to see what (and why) it's happening?

<img width="1173" height="178" alt="Screenshot 2025-10-15 at 3 26 28 PM" src="https://github.com/user-attachments/assets/43dfb91d-e7a7-47ae-aec6-9fc8b88aaf08" />

Reply:
The logging for this script is massively improved in the next PR for the Typescript SDK.